seam/request.py

```diff
@@ -0,0 +1,47 @@
+import requests
+from importlib.metadata import version
+
+from seam.types import AbstractRequestMixin, SeamApiException
+
+
+class RequestMixin(AbstractRequestMixin):
+    def make_request(self, method: str, path: str, **kwargs):
+        """
+        Makes a request to the API
+
+        Parameters
+        ----------
+        method : str
+          Request method
+        path : str
+          Request path
+        **kwargs
+          Keyword arguments passed to requests.request
+
+        Raises
+        ------
+        SeamApiException: If the response status code is not successful.
+        """
+
+        url = self._endpoint + path
+        sdk_version = version("seam")
+        headers = {
+            **self._auth_headers,
+            "Content-Type": "application/json",
+            "User-Agent": "Python SDK v"
+            + sdk_version
+            + " (https://github.com/seamapi/python-next)",
+            "seam-sdk-name": "seamapi/python",
+            "seam-sdk-version": sdk_version,
+            "seam-lts-version": self.lts_version,
+        }
+
+        response = requests.request(method, url, headers=headers, **kwargs)
+
+        if response.status_code != 200:
+            raise SeamApiException(response)
```

Comment:
Should we name this `SeamHttpApiError`? https://github.com/seamapi/javascript-http/blob/main/src/lib/seam/connect/seam-http-error.ts

It would match `SeamActionAttemptTimeoutError` etc. Python seems to have a convention of naming a custom `Exception` with `Error` 🤷🏻

Reply:
Yep, good point. Will update it once only route files are generated (relevenat PR https://github.com/seamapi/python-next/pull/47) because otherwise it'll get rewritten by the generator. 
Created appropriate issue https://github.com/seamapi/python-next/issues/48.